cnf-certification-test/accesscontrol/tolerations/tolerations.go

```diff
@@ -40,12 +41,24 @@ func IsTolerationModified(t v1.Toleration) bool {
 	//   key: node.kubernetes.io/unreachable
 	//   operator: Exists
 	//   tolerationSeconds: 300
+	// # this last one, only if QoS class for the pod is different than BestEffort
+	// - effect: NoSchedule
+	//   key: node.kubernetes.io/memory-pressure
+	//   operator: Exists
 
 	if t.Effect == v1.TaintEffectNoExecute {
 		if t.Key == notReadyStr || t.Key == unreachableStr &&
 			(t.Operator == v1.TolerationOpExists && t.TolerationSeconds != nil && *t.TolerationSeconds == int64(tolerationSecondsDefault)) {
 			return false
 		}
+	} else if t.Effect == v1.TaintEffectNoSchedule {
+		// If toleration is NoSchedule - node.kubernetes.io/memory-pressure - Exists and the QoS class for
+		// the pod is different than BestEffort, it is also a default toleration added by k8s
+		if (t.Key == memoryPressureStr) &&
+			(t.Operator == v1.TolerationOpExists && t.TolerationSeconds == nil) &&
```

Comment:
I don't think the `t.TolerationSeconds == nil` check is needed here. As per k8s docs, that field is only used for NoExecute effects.

Reply:
let's remove that, then, I just included it to make sure it is not defined, but if it's not related to NoSchedule, let's remove it